analysis/plotScores.py

```diff
@@ -0,0 +1,69 @@
+import argparse
+import csv
+import matplotlib.pyplot as plt
+
+def parse_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('input_csv', type=str,
+                        help='a CSV file (with headers) containing the scores for each subject')
+    return parser.parse_args()
+
+# Makes TWO dictionaries
+# subjDict is a dictionary of the scores for EACH subject
+# Key: Listener ID
+# Value: Dictionary (Key: SNR; Value: List of scores)
+# allDict is a dictionary of the scores for ALL subjects
+# Key: SNR
+# Value: List of scores
```

Comment:
Usually, the description of how `makeDicts` works goes *inside* the definition (and is indented) rather than right before it.